### DISCUSSION TITLE
Earnapp url active

### DISCUSSION THREAD

**Discussion by jupitert25** [engageub/InternetIncome#64] [Q&A]
Hello. Thank you for the scripts you made 
I have a problem. I don't know how to add the device Earnapp to the dashboard
I read your scripts and still can't do it. so I using proxies. How can I open the URL to connect the device to Earnapp? Do I need to install Ebesucher for Firefox? and how can I open Firefox when the host using a proxy? . Since I can't connect vps with localhost port 2000

**Comment by engageub** [comment]
Hi,
You have to copy the urls present in earnapp.txt and paste in the browser after logging into your earnapp dashboard.
Its a regular process that you do after installing earnapp. 

Ebesucher and earnapp are different. 

Please go through FAQ to open firefox. But its not required as the process is automated. 

https://github.com/engageub/InternetIncome#grey_question-faq

Thank you

**Reply by jupitert25** [reply]
I using a proxy so each folder will be my proxy http://ip: port 2000 or my VPS IP . 
and I tried both still can't open it.

**Reply by engageub** [reply]
Hi,
You can use multiple proxies in same folder by adding it to proxies.txt file.
If your VPS has port restrictions, please use serveo command to access the browser. 
You will find the urls in ebesucher.txt file. 

```
ssh -R 80:localhost:2000 serveo.net
```

Thank you

**Reply by jupitert25** [reply]
it's working well now . thank you . I start doing Mysterium and it shwoing Monitoring failed .  I running proxy so maybe I got error u mention about it in FAQ . So Is there anyway to fix this? I try to follow Mysterium Firewall (iptables) configuration but not working

**Reply by engageub** [reply]
Hi,
Mysterium works with openvpn but not proxies at the moment. The ongoing issue is mentioned in FAQ. 
If you wish to use multi IP or openvpn, please use the test branch.

https://github.com/engageub/InternetIncome/tree/test

Thank you
